pkg/controllers/subnetbinding/subnetbinding_controller.go

```diff
@@ -208,36 +209,94 @@ func (r *Reconciler) listBindingMapIDsFromCRs(ctx context.Context) (sets.Set[str
 //     Subnet/SubnetSet readiness update will actively trigger a requeue event
 //  2. the associated Subnet is already used as a target Subnet in another SubnetConnectionBindingMap CR, or the target
 //     Subnet already has associated SubnetConnectionBindingMap CR. In this case, a retry error is returned.
+//  3. the target Subnet is a pre-created Subnet. In this case, a not-retry error is returned.
+//  4. the target Subnet and associated Subnet are not in the same vpc. In this case, not-retry error is returned.
```

Comment:
Do you mean the target Subnet CR and associated Subnet CR are located in the same K8s Namespace, but the corresponding NSX VpcSubnets arr in the different VPCs?